apps/store/extensions/app/store-common/pages/top_assets.jag

```diff
@@ -65,8 +66,9 @@ require('/modules/store.js').exec(function (ctx) {
         }
     }
     var qtemp = ctx.request.getParameter('q') || '';
-    var qjson = parse('{'+qtemp+'}');
-    output.searchQuery = qjson.name||'';
+    var searchQuery = assetApi.formatSearchQuery(qtemp);
+    output.searchQuery = searchQuery||'';
```

Comment:
Thanks @mushthaq33 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/1cfd4a532c0b6fcbb158ab5da98f004afac2612c)